geopandas/geoseries.py

```diff
@@ -717,26 +706,30 @@ def _constructor_expanddim_from_mgr(self, mgr, axes):
         return _expanddim_logic(df)
 
     def _wrapped_pandas_method(self, mtd, *args, **kwargs):
-        """Wrap a generic pandas method to ensure it returns a GeoSeries"""
+        """Wrap a generic pandas method to ensure it returns a GeoSeries."""
         val = getattr(super(), mtd)(*args, **kwargs)
         if type(val) is Series:
             val.__class__ = GeoSeries
             val.crs = self.crs
         return val
 
     def __getitem__(self, key):
+        """Get item from GeoSeries."""
         return self._wrapped_pandas_method("__getitem__", key)
 
     @doc(pd.Series)
     def sort_index(self, *args, **kwargs):
+        """Sort by labels (along an axis)."""
```

Comment:
We're getting the docstring from pandas here. not sure what happens if you specify one explicitly. Same below.